Packs/ArcusTeam/Integrations/ArcusTeam/ArcusTeam.py

```diff
@@ -0,0 +1,202 @@
+import json
+import requests
+import urllib3
+import traceback
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+from typing import Any, Dict
+
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+""" CONSTANTS """
+VERSION = "1.0.0"
+
+
+""" CLIENT CLASS """
+
+
+class Client(BaseClient):
+
+    def get_devices(self, vendor, model, series, firmware_version):
+
+        return self._http_request(
+            method='POST',
+            url_suffix='/get_devices',
+            json_data={
+                "vendor": vendor,
+                "model": model,
+                "series": series,
+                "firmware_version": firmware_version,
+                "version": VERSION
+            })
+
+    def get_vulnerabities(self, firmwareId, deviceId, pageSize, page, sortField, sortOrder, returnFields):
+
+        return self._http_request(
+            method='POST',
+            url_suffix='/get_vulnerabilities',
+            json_data={
+                "firmwareId": firmwareId,
+                "version": VERSION,
+                "deviceId": deviceId,
+                "pageSize": pageSize,
+                "page": page,
+                "sortOrder": sortOrder,
+                "sortField": sortField,
+                "returnFields": returnFields,
+            })
+
+
```

Comment:
move authentication function inside client class
```suggestion
    def get_authentication_token(self, authentication_url: str, headers: Dict, payload: Dict):
        return self._http_request(
            method='POST',
            full_url=authentication_url,
            headers=headers,
            data=payload
        )

```